imgui-examples/Cargo.toml

```diff
@@ -16,3 +16,7 @@ image = "0.23"
 imgui = { path = "../imgui" }
 imgui-glium-renderer = { path = "../imgui-glium-renderer" }
 imgui-winit-support = { path = "../imgui-winit-support" }
+
+# Direct dependency only required because `winit::VirtualKeyCode` is
+# used for one example.
+winit = "*"
```

Comment:
glob import will be a problem here... kind of a pain to fix, for now please just hardcode the constant winit uses for it.

Reply:
Not sure I understand - you mean pin this to, say, `winit = "0.24.0"` ?

Reply:
No, remove it. It will prevent us from actually running tests on different winit versions. The way version resolution works, this isn't actually a wildcard like you'd think.

I think we have two choices here: either we figure out what value `VirtualKeyCode::Home as i32` is and hardcode that, or reexport winit from imgui-winit-support. My gut was to do the first one, but the 2nd is probably reasonable too, although probably I'd prefer we `#[doc(hidden)]` it, since people should probably depend on it explicitly.

Reply:
Ahh, I see! That makes sense. I'll just hardcode it - if we use the re-export in examples hiding in docs wont stop copy-pasting!